src/content/developers/docs/nodes-and-clients/index.md

```diff
@@ -171,6 +172,14 @@ Nethermind is an Ethereum implementation created with the C# .NET tech stack, li
 
 Nethermind also has [detailed documentation](https://docs.nethermind.io), strong dev support, an online community and 24/7 support available for premium users.
 
+### Ethereumjs {#ethereumjs}
+
+It is written in TypeScript and composed of a number of packages, including core Ethereum primitives represented by the the Block, Transaction, and Merkle-Patricia Trie classes and core client components including an implementation of the Ethereum Virtual Machine (EVM), a blockchain class, and the DevP2P networking stack.
```

Comment:
It -> "The EthereumJS Client"

Reply:
Update: or maybe "The EthereumJS Execution Client (EthereumJS)" here, and the "It" for the next paragraph below.

Reply:
Fixed.